package.json

```diff
@@ -94,7 +95,9 @@
     "react-dom": "^16.13.1",
     "react-redux": "^7.2.0",
     "react-router-dom": "^5.2.0",
-    "redux": "^4.0.5"
+    "redux": "^4.0.5",
+    "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.3.0"
```

Comment:
он уже есть внутри redux-toolkit

Reply:
https://redux-toolkit.js.org/api/getDefaultMiddleware

Reply:
https://redux-toolkit.js.org/api/getDefaultMiddleware#development 

> In addition to these development tool middleware, it also adds redux-thunk by default, since thunks are the basic recommended side effects middleware for Redux.

Reply:
Я больше этим хочу проиллюстрировать как добавлять с нуля, тулкита может и не быть